### DISCUSSION TITLE
What is the default Password for Admin or do we have to create the admin account by signup?

### DISCUSSION THREAD

**Discussion by amanpal108** [HiEventsDev/Hi.Events#161] [Q&A]
I can'r find any login credentials for admin account in non-saas mode

**Comment by daveearley** [comment]
There is no default password or account. You will need to create an account before you can login.

**Reply by StefanusHosea** [reply]
I've deployed via Digital Ocean. But when I tried to register an account, I'm stuck with "An unexpected error occurred. Please try again."

The runtime logs showing:
`23/Aug/2024:19:56:49 +0000 "OPTIONS /index.php/auth/register" 204`

I've tried to add on the environment variables:
VITE_FRONTEND_URL : ${APP_URL}
VITE_API_URL_CLIENT : ${APP_URL}/api
VITE_API_URL_SERVER : http://localhost:80/api
APP_CDN_URL : ${APP_URL}/storage
APP_FRONTEND_URL : ${APP_URL}

Please help.

**Reply by daveearley** [reply]
Are there any other errors in the log? Did you follow these steps [here](https://github.com/HiEventsDev/Hi.Events-digitalocean?tab=readme-ov-file#-how-to-use)? Including setting an APP_KEY?

**Reply by StefanusHosea** [reply]
Yes, I've set the APP_KEY and JWT_SECRET and checked the "Encrypt" checkbox as follows:

![image](https://github.com/user-attachments/assets/d2028801-d45e-4e96-9ad5-04dc25646d1c)

The error in my network tab is
```
Request URL: https://domain.tld/api/auth/register
Request Method: OPTIONS
Status Code: 204 No Content
```
![image](https://github.com/user-attachments/assets/05f037b9-a3fc-4394-8e25-e223746aeb1f)

**Reply by daveearley** [reply]
This looks like a CORS issue. Try updating `APP_FRONTEND_URL` in DigitalOcean to match your domain name.

**Reply by FourHeadedDev** [reply]
i am getting the same issue and am unable to fix that if anybody get soln please\help
